kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
```

Comment:
I don't think we can if we're integrating this with django-pipeline and need to use staticfiles_storages? But, I can use `get_path` which `statici18n` just wraps.